v2/session/serverless/with-aws-lambda/setup-api-gateway.mdx

```diff
@@ -57,8 +60,8 @@ Click on the `{proxy+}` resource and then "Enable CORS" button to open the CORS
 
 Click the **Deploy API** button in the top right corner to deploy the API. During deployment, you'll be prompted to create a stage; for this tutorial, name the stage `dev`. After deployment, you will receive your `Invoke URL`.
 
-:::note
-Update `apiBasePath` and `apiGatewayPath` in the Lambda configuration if they have changed post API Gateway configuration.
+:::important
+Update `apiDomain`, `apiBasePath`, and `apiGatewayPath` in both Lambda configuration and your frontend config if they have changed post API Gateway configuration.
```

Comment:
The apiGatewayPath is not a frontend config. So split this into two points - one for backend update, and one for frontend.

Reply:
It is there in [supertokens-auth-react](https://github.com/supertokens/supertokens-auth-react/blob/51976b8331cd021e31485dab7a37cb859bc9a83d/lib/ts/types.ts#L152).